spec/spec.md

```diff
@@ -203,6 +203,28 @@ This profile uses certain versions of specifications that have not yet reached f
   - First Implementer's Draft of OpenID for Verifiable Prensetations specification
   - ID Hub specification published as a v0.0.1 predraft of [Decentralized Web Node](https://identity.foundation/decentralized-web-node/spec/). We will continue to use the term ID Hub rather than Decentralized Web Node to avoid confusion.
 
+### JWT-VCs
```

Comment:
```suggestion
### JWT VCs
```

we don't use a dash between them anywhere else.

Reply:
can we do the opposite and start using dashes?

Reply:
Sure... should we add a dash to the profile game too?
"JWT-VC Presentation Profile"